frontend/web/components/pages/AuditLogPage.tsx

```diff
@@ -3,7 +3,7 @@ import ConfigProvider from 'common/providers/ConfigProvider';
 import ToggleChip from '../ToggleChip';
 import Utils from 'common/utils/utils';
 import { Project } from 'common/types/responses';
-import { RouterChildContext } from 'react-router';
+import { RoutPanelSearcherChildContext } from 'react-router';
```

Comment:
`RoutPanelSearcherChildContext` seems like a typo? Perhaps not though, maybe it's normal to shorten to `Rout`